orga/tests/acceptance/join-test.js

```diff
@@ -312,4 +313,117 @@ module('Acceptance | join', function(hooks) {
       });
     });
   });
+
+  module('When user is already connected', function(hooks) {
+
+    let user;
+    let code;
+    let organizationId;
+
+    hooks.beforeEach(async () => {
+      code = 'ABCDEFGH01';
+      organizationId = server.create('organization', { name: 'College BRO & Evil Associates' }).id;
+      server.create('campaign');
+
+      user = createUserWithMembershipAndTermsOfServiceAccepted();
+      await authenticateSession({
+        user_id: user.id,
+        access_token: 'aaa.' + btoa(`{"user_id":${user.id},"source":"pix","iat":1545321469,"exp":4702193958}`) + '.bbb',
+        expires_in: 3600,
+        token_type: 'Bearer token type',
+      });
+
+      await visit('/');
+    });
+
+    test('it should go on join page when organization-invitation is pending', async function(assert) {
+      // given
+      const organizationInvitationId = server.create('organizationInvitation', {
+        organizationId, email: 'random@email.com', status: 'pending', code
+      }).id;
+
+      const expectredURL = `/rejoindre?code=${code}&invitationId=${organizationInvitationId}`;
+
+      // when
+      await visit(`/rejoindre?invitationId=${organizationInvitationId}&code=${code}`);
+
+      // then
+      assert.equal(currentURL(), expectredURL);
+      assert.notOk(currentSession(this.application).get('isAuthenticated'), 'The user is still unauthenticated');
+    });
+
+    test('it should redirect to login page when organization-invitation does not exist', async function(assert) {
+      // when
+      await visit('rejoindre?invitationId=123456&code=FAKE999');
+
+      // then
+      assert.equal(currentURL(), '/connexion');
+      assert.notOk(currentSession(this.application).get('isAuthenticated'), 'The user is still unauthenticated');
+    });
+
+    test('it should redirect to login page when organization-invitation has already been accepted', async function(assert) {
```

Comment:
pour la cohérence et la facilité de lecture, je propose
```suggestion
    module('when organization-invitation has already been accepted', function () {
      test('it should redirect to login page', async function(assert) {
```

Reply:
Done